api/grpc/v1/agent.proto

```diff
@@ -0,0 +1,67 @@
+syntax = "proto3";
+
+package flightctl.v1;
+
+option go_package = "github.com/flightctl/flightctl/api/grpc/v1/grpc-v1";
```

Comment:
Should this be structured by purpose (`/api/agent`) rather than technology (`/api/grpc`)?
Along the same lines: Should the user API also move to a subdir?
I'm thinking we may need to collect more APIs here at some point, e.g. the Config APIs for the different services.